boards/arm/cc1352r_sensortag/support/openocd.cfg

```diff
@@ -0,0 +1,8 @@
+#
+# TI CC1352R SensorTag
+#
+source [find interface/jlink.cfg]
```

Comment:
Is there a way that I can remove this line so that the interface is not hard-coded?

I tried something like
```
west flash --runner openocd --cmd-pre-init "source [find interface/jlink.cfg]"
```
but that doesn't work.

I guess that the best solution is to copy / modify the config file and then use `--config CONFIG`.

Might be a nice feature in the future to preprocess the openocd.cfg file and prepend an interface line based on an argument to west flash. E.g. the `--iface` argument to jlink could be recycled for openocd for this purpose.